seeds.sql

```diff
@@ -1,11 +1,2532 @@
+(31, 'user_31@idp.com', 10, 'user31@example.com', 'User 31', 'NONE', '2025-02-08 00:00:00', '2025-02-14 00:00:00', 'Web'),
+(32, 'user_32@idp.com', 2, 'user32@example.com', 'User 32', 'NONE', '2025-02-03 00:00:00', '2025-02-16 00:00:00', 'Web'),
+(33, 'user_33@idp.com', 4, 'user33@example.com', 'User 33', 'NONE', '2025-02-15 00:00:00', '2025-03-01 00:00:00', 'Web'),
+(34, 'user_34@idp.com', 1, 'user34@example.com', 'User 34', 'NONE', '2025-02-07 00:00:00', '2025-02-17 00:00:00', 'Web'),
+(35, 'user_35@idp.com', 7, 'user35@example.com', 'User 35', 'NONE', '2025-02-24 00:00:00', '2025-03-13 00:00:00', 'Web'),
+(36, 'user_36@idp.com', 2, 'user36@example.com', 'User 36', 'NONE', '2025-02-22 00:00:00', '2025-02-25 00:00:00', 'Web'),
+(37, 'user_37@idp.com', 5, 'user37@example.com', 'User 37', 'NONE', '2025-02-18 00:00:00', '2025-02-24 00:00:00', 'Web'),
+(38, 'user_38@idp.com', 9, 'user38@example.com', 'User 38', 'NONE', '2025-02-23 00:00:00', '2025-03-06 00:00:00', 'Web'),
+(39, 'user_39@idp.com', 1, 'user39@example.com', 'User 39', 'NONE', '2025-03-03 00:00:00', '2025-03-15 00:00:00', 'Web'),
+(40, 'user_40@idp.com', 6, 'user40@example.com', 'User 40', 'NONE', '2025-03-03 00:00:00', '2025-03-04 00:00:00', 'Web'),
+(41, 'user_41@idp.com', 5, 'user41@example.com', 'User 41', 'NONE', '2025-02-11 00:00:00', '2025-02-16 00:00:00', 'Web'),
+(42, 'user_42@idp.com', 6, 'user42@example.com', 'User 42', 'NONE', '2025-02-04 00:00:00', '2025-02-15 00:00:00', 'Web'),
+(43, 'user_43@idp.com', 3, 'user43@example.com', 'User 43', 'NONE', '2025-02-14 00:00:00', '2025-02-26 00:00:00', 'Web'),
+(44, 'user_44@idp.com', 6, 'user44@example.com', 'User 44', 'NONE', '2025-02-15 00:00:00', '2025-02-22 00:00:00', 'Web'),
+(45, 'user_45@idp.com', 9, 'user45@example.com', 'User 45', 'NONE', '2025-02-02 00:00:00', '2025-02-14 00:00:00', 'Web'),
+(46, 'user_46@idp.com', 2, 'user46@example.com', 'User 46', 'NONE', '2025-02-20 00:00:00', '2025-03-11 00:00:00', 'Web'),
+(47, 'user_47@idp.com', 3, 'user47@example.com', 'User 47', 'NONE', '2025-02-16 00:00:00', '2025-03-06 00:00:00', 'Web'),
+(48, 'user_48@idp.com', 10, 'user48@example.com', 'User 48', 'NONE', '2025-02-10 00:00:00', '2025-02-23 00:00:00', 'Web'),
+(49, 'user_49@idp.com', 7, 'user49@example.com', 'User 49', 'NONE', '2025-02-10 00:00:00', '2025-02-17 00:00:00', 'Web'),
+(50, 'user_50@idp.com', 9, 'user50@example.com', 'User 50', 'NONE', '2025-02-26 00:00:00', '2025-03-15 00:00:00', 'Web');
 
-INSERT INTO `user` (`id`, `user_id_from_idp`, `idp_id`, `email`, `fullname`, `adminLevel`, `created_at`, `last_login_at`, `last_login_platform`)
-VALUES
-	(-21, 'dummy@toadreader.com', 21, 'dummy@toadreader.com', 'No login', 'NONE', '2020-10-14 16:47:50.859', '2024-01-23 17:27:06.070', NULL);
+INSERT INTO `book` (`title`, `author`, `isbn`, `coverHref`, `rootUrl`, `updated_at`, `standardPriceInCents`, `epubSizeInMB`, `audiobookInfo`) VALUES
```

Comment:
Booklane have their own ideas about how audiobooks should work. We don't use their implementation, so that column can probably be left empty.

Personally I would have us upload epubs when we start the instance, but if you want to prepolute books please use books that either have expired copyright or were never copyrighted, e.g. [Project Gutenberg](https://www.gutenberg.org/).

Please do not list any of the epubs that have been provided for testing. This is currently the case, but I just want to be sure that this will continue to be the case